src/test/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyProfileServiceImplTest.java

```diff
@@ -202,4 +204,110 @@ void deleteNoCompanyProfile() {
         verify(repository, never()).delete(any());
     }
 
+    @Test
+    void createDissolvedCompany() {
```

Comment:
I think it might no longer make sense to keep the existing test methods `createEnglandWales` and `createScotland`? Maybe we could just have different jurisdictions in the more relevant tests now?

I'd say we need tests for:
- dissolved (as it will also have insolvency history) - this one ✅ 
- empty/non-empty company type
- empty/non-empty company status